pom.xml

```diff
@@ -119,6 +119,12 @@
 				<version>${legacyuiVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
```

Comment:
Do we need this now?

Reply:
yes, this decency is needed for java 17

Reply:
You could also add this to the Java 17 pull request.